disputer/package.json

```diff
@@ -4,7 +4,9 @@
   "description": "UMA Disputer",
   "private": true,
   "dependencies": {
-    "dotenv": "^6.2.0",
+    "dotenv": "^6.2.0"
+  },
+  "devDependencies": {
```

Comment:
Is Winston a dev dependency?

Reply:
Oh good point. It's not

Reply:
Isn't it, though? I only see `require('winston')` in tests